modules/sdk/project-templates/project-templates-activator/src/main/resources/archetype-resources/bnd.bnd

```diff
@@ -1,4 +1,6 @@
 Bundle-Activator: ${package}.${className}
 Bundle-Name: ${artifactId}
 Bundle-SymbolicName: ${package}
-Bundle-Version: ${version}
\ No newline at end of file
+Bundle-Version: ${version}#if (${buildType} != "gradle")
+
+-metayype: *#end
```

Comment:
Should be `-metatype: *`

Reply:
O_o,  darn small font. okay I'll fix it and send it back